lib/timeline/Core.js

```diff
@@ -654,10 +648,12 @@ Core.prototype.addCustomTime = function (time, id, options) {
     throw new Error('A custom time with id ' + JSON.stringify(id) + ' already exists');
   }
 
-  var customTime = new CustomTime(this.body, util.extend({}, this.options, options, {
+  var customTime = new CustomTime(this.body, {
+    ...this.options,
+    ...options,
     time : timestamp,
```

Comment:
What if `this.options` or `option` also have a `time` or a `id`? Not sure what will happen here...

Reply:
It will take the `time` and `id`. That's how destructuring works.